src/pages/DashboardPage/index.js

```diff
@@ -0,0 +1,68 @@
+import React, { useEffect } from 'react';
+import { useSelector, useDispatch } from 'react-redux';
+
+import './DashboardPage.scss';
+import smilies from 'assets/smilies.svg';
+import defaultProfile from 'assets/default_profile.svg';
+import football from 'assets/football.svg';
+import music from 'assets/music.svg';
+import world from 'assets/world.svg';
+
+import { getProfile } from 'redux/actions/userActions';
+
+const WelcomePage = () => {
+  const user = useSelector(({ user: { value } }) => value);
+
+  const dispatch = useDispatch();
+  useEffect(() => {
+    dispatch(getProfile(user?.id));
```

Comment:
Maybe: 
```suggestion
    user && dispatch(getProfile(user.id));
```
You don't want to bother the API with an `/undefined` request.